src/inference/include/openvino/runtime/intel_gna/properties.hpp

```diff
@@ -271,5 +273,11 @@ static constexpr Property<PWLDesignAlgorithm> pwl_design_algorithm{"GNA_PWL_DESI
  */
 static constexpr Property<float> pwl_max_error_percent{"GNA_PWL_MAX_ERROR_PERCENT"};
 
+/**
+ * @brief By default gna api works with Int16 weights precision, however this can be adjusted if necessary,
+ * currently supported values are I16, I8
+ */
+static constexpr Property<InferenceEngine::Precision> precision{"GNA_PRECISION"};
```

Comment:
don't use API 1.0 entries in API 2.0

why OpenVINO's `inference_precision` does not work for GNA?

Reply:
Is your intention to remove GNA_PRECISION property from GNA Plugin and use just inference_precision?

Reply:
Ok, this was misunderstanding. I didn't want to extend OV API. 
I've moved tests to local set, so this property is not needed anymore.